.github/workflows/ci.yml

```diff
@@ -31,15 +31,14 @@ jobs:
 
       - name: Setup CodeQL
         if: steps.changes.outputs.src == 'true'
+        id: install-codeql
         uses: ./.github/actions/install-codeql
-        with:
```

Comment:
Also remove the version number from the other invocations of the `install-codeql`